src/dev_tools/makeClass/templates/blank_options.go.tmpl

```diff
@@ -77,3 +77,10 @@ func GetOptions() *[{PROPER}]Options {
 	// EXISTING_CODE
 	return &default[{PROPER}]Options
 }
+
+func ResetOptions() {
```

Comment:
Question for you: do you know how to run the auto code generator?

Reply:
I think I do. Something along the lines of z`makeClass" "--options --gocmds`?

Reply:
If it's configured, it should run every time you run either `make test-all` or `test-all.sh`. To configure, add this section to a file called `~/.local/share/trueblocks/makeClass.toml`.

```
[enabled]
generate = true
auto_format = true
readmes = true
```